libraries/AC_WPNav/AC_WPNav_OA.cpp

```diff
@@ -1,3 +1,7 @@
+#include <AC_Avoidance/AC_Avoid_config.h>
+
+#if AP_OAPATHPLANNER_ENABLED
```

Comment:
```suggestion
#include "AC_WPNav_config.h"

#if AC_WPNAV_OBJECTAVOIDANCE_ENABLED
```

... and define it based on AC_OAPATHPLANNER_ENABLED by default

Reply:
Done. Thanks.